pglast/printers/ddl.py

```diff
@@ -878,6 +878,41 @@ def create_function_option(node, output):
     output.print_symbol(node.arg)
 
 
+@node_printer('CreateOpClassStmt')
+def create_opclass_stmt(node, output):
+    output.write('CREATE OPERATOR CLASS ')
+    output.print_name(node.opclassname)
+    if node.isDefault:
+        output.write('DEFAULT ')
+    output.write('FOR TYPE ')
+    output.print_name(node.datatype)
+    output.write('USING ')
+    output.print_name(node.amname)
+    output.write('AS ')
+    output.print_list(node.items, ',')
+
+
+@node_printer('CreateOpClassItem')
+def create_opclass_item(node, output):
+    unquote_name = False
+    if node.itemtype == 1:
```

Comment:
I'd prefer using the symbolic constants here: https://github.com/lelit/pglast/blob/master/pglast/enums/parsenodes.py#L432

Reply:
Done ! Sorry, I missed those.